src/ucp/tag/rndv.c

```diff
@@ -512,6 +512,8 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_rndv_progress_rma_get_zcopy, (self),
         ucp_rkey_destroy(rndv_req->send.rndv_get.rkey);
         ucp_rndv_recv_data_init(rndv_req->send.rndv_get.rreq,
                                 rndv_req->send.length);
+        UCP_WORKER_STAT_RNDV(ep->worker, GET_ZCOPY, -1);
+        UCP_WORKER_STAT_RNDV(ep->worker, RTR,       +1);
```

Comment:
we increase `RX_RNDV_RTR` counter before to **send** RTR. Looks confusing,
also there are other places where we send rtr, maybe increase stats in `ucp_rndv_req_send_rtr`?

Reply:
`ucp_rndv_req_send_rtr` is also used from fragment-rtr and we don't want to count those

Reply:
what is about naming? RTR counter reflects sending of `RTR`, but it's name is `RX_RNDV_RTR`